testing-modules/junit-5/pom.xml

```diff
@@ -105,21 +105,6 @@
             </resource>
         </resources>
         <plugins>
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>exec-maven-plugin</artifactId>
-                <version>${exec-maven-plugin.version}</version>
-                <executions>
-                    <execution>
-                        <goals>
-                            <goal>java</goal>
-                        </goals>
-                    </execution>
-                </executions>
-                <configuration>
-                    <mainClass>com.baeldung.TestLauncher</mainClass>
```

Comment:
I think we should keep this explicit definition in the junit-5 module, as this is the only one that has a TestLauncher class, for this article: https://www.baeldung.com/junit-5-extensions 
and remove the configuration from the junit-5-migration and test-containers modules

Reply:
Yeah, that sounds good @lor6 Thanks for catching that!